lib/CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 include(GNUInstallDirs)
 
-add_library(mtkahypar SHARED libmtkahypar.cpp)
+add_library(mtkahypar libmtkahypar.cpp)
 target_link_libraries(mtkahypar ${Boost_LIBRARIES} TBB::tbb TBB::tbbmalloc_proxy)
```

Comment:
If you build mt-kahypar as a static library. TBB must be also compiled as a static library:
https://stackoverflow.com/questions/638278/how-to-statically-link-to-tbb
Otherwise, this will not work.